.gitignore

```diff
@@ -7,6 +7,7 @@
 
 log/*
 csv_downloads/*
+tmp/*
```

Comment:
Should add zipfiles  under /downloads to the .gitignore so when people in development are testing download it gets tracked in git as a new file

Reply:
Thanks, good suggestion.  Actually, I notice that `csv_downloads` was already in .gitignore and earlier code was saving csvs to there, which seems like a better name, so I just changed this code to save there.